lints/base.go

```diff
@@ -207,6 +207,11 @@ type LintReport struct {
 	ERsaNoPublicKey                                      ResultStruct `json:"e_rsa_no_public_key,omitempty"`
 	ESubCertCertificatePoliciesMissing                   ResultStruct `json:"e_sub_cert_certificate_policies_missing,omitempty"`
 	ESubCertKeyUsageCrlSignBitSet                        ResultStruct `json:"e_sub_cert_key_usage_crl_sign_bit_set,omitempty"`
+	InfosPresent                                         bool         `json:"infos_present,omitempty"`
```

Comment:
_sigh_, Yeah, I agree. The issue I was running into is that the results are abstracted away from the `ZLintResult` struct. I think what I can do is have the `LintReport` keep track of whether or not it has an error or warning or wahtever, and then bubble that up to a field in `ZLintResult`